Microsoft.DotNet.Interactive.PowerShell/Host/VTColorUtils.cs

```diff
@@ -9,6 +9,7 @@ internal static class VTColorUtils
     {
         internal const string EscapeCharacters = "\x1b[";
         internal const string ResetColor = "\x1b[0m";
+        internal const string BoldColor = "\x1b[1m";
```

Comment:
We've added a reference to `System.CommandLine.Rendering` so we might want to refactor to use that for VT. I'm happy to help with the the refactor, as I'd love feedback on the APIs.

Reply:
Would love to refactor and use that library! Opened #299 to track the work.